packages/avatars/src/elements/StatusIndicator.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useText } from '@zendeskgarden/react-theming';
+import ClockIcon12 from '@zendeskgarden/svg-icons/src/12/clock-stroke.svg';
+import ClockIcon16 from '@zendeskgarden/svg-icons/src/16/clock-stroke.svg';
+import ArrowLeftIcon12 from '@zendeskgarden/svg-icons/src/12/arrow-left-sm-stroke.svg';
+import ArrowLeftIcon16 from '@zendeskgarden/svg-icons/src/16/arrow-left-sm-stroke.svg';
+
+import { IStatusIndicatorProps, STATUS } from '../types';
+import {
+  StyledStandaloneStatusIndicator,
+  StyledStandaloneStatusCaption,
+  StyledStandaloneStatus
+} from '../styled';
+
+/**
+ * @extends HTMLAttributes<HTMLElement>
+ */
+const StatusIndicatorComponent = forwardRef<HTMLElement, IStatusIndicatorProps>(
+  ({ children, type, isCompact, ...props }, ref) => {
+    let ClockIcon = ClockIcon16;
+    let ArrowLeftIcon = ArrowLeftIcon16;
+
+    if (isCompact) {
+      ClockIcon = ClockIcon12;
+      ArrowLeftIcon = ArrowLeftIcon12;
+    }
+
+    const defaultLabel = useMemo(() => ['status'].concat(type || []).join(': '), [type]);
+    const ariaLabel = useText(StatusIndicatorComponent, props, 'aria-label', defaultLabel);
+
+    return (
+      <StyledStandaloneStatus role="status" ref={ref} {...props}>
+        <StyledStandaloneStatusIndicator
+          role="img"
+          type={type}
+          isCompact={isCompact}
+          aria-label={ariaLabel}
+        >
+          {type === 'away' ? <ClockIcon data-icon-status={type} aria-hidden="true" /> : null}
+          {type === 'transfers' ? (
+            <ArrowLeftIcon data-icon-status={type} aria-hidden="true" />
+          ) : null}
+        </StyledStandaloneStatusIndicator>
```

Comment:
```suggestion
        <StyledStatusIndicator
          role="img"
          status={type}
          aria-label={ariaLabel}
          size={isCompact ? 'small' : 'medium'}
        >
          {type === 'away' ? <ClockIcon data-icon-status={type} aria-hidden="true" /> : null}
          {type === 'transfers' ? (
            <ArrowLeftIcon data-icon-status={type} aria-hidden="true" />
          ) : null}
        </StyledStatusIndicator>
```

If you try this suggestion, I think you'll see that the new `StyledStandaloneStatusIndicator` is almost entirely unnecessary – with one exception. It should only add a `margin-top` property. The pseudo-code for `margin-top` is:

`margin-top: calc((${lineHeight} - ${iconSize}) / 2);`

Reply:
Unfortunately, `box-shadow` is another visual property that is noticeable when the background does not match the color of the  `box-shadow`. Where do you propose this line (`margin-top: calc((${lineHeight} - ${iconSize}) / 2)`) to exist?

Reply:
The point of the suggestion is to show how close the existing styling gets us. I think the definition of `StyledStandaloneStatusIndicator` should look something like this:

```ts
export const StyledStandaloneStatusIndicator = styled(StyledStatusIndicator).attrs({ ... })`
  margin-top: calc((${lineHeight} - ${iconSize}) / 2);
  box-shadow: none;

  ${props => retrieveStyledComponents(...)};
`;
```